config/environment.js

```diff
@@ -2,5 +2,8 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return {};
+  const ENV = {
+    productionEnvironments: ['prod'],
+  };
+  return ENV;
```

Comment:
We don't want to have force our internal apps to provide the environment addition. 
This file _should_ have no effect on the apps that use it.
Most addons will handle defaults like [this](https://github.com/poteto/ember-cli-flash/blob/652412bdb9a025b803f8dd22d8c894fd3761e2e7/app/initializers/flash-messages.js#L10).